atproto/lexicon/cmd/lextool/net.go

```diff
@@ -29,12 +28,12 @@ func runValidateRecord(cctx *cli.Context) error {
 	cat := lexicon.NewBaseCatalog()
 	err := cat.LoadDirectory(p)
 	if err != nil {
-		return err
+		return errors.Wrap(err, "in LoadDirectory")
```

Comment:
Especially when a function has multiple error return sites, it's useful to wrap them to know which site did the returning.